holidayMealBagForm.php

```diff
@@ -1,65 +1,103 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="css/base.css">
-    <title>Holiday Meal Bag 2023</title>
-</head>
-<body>
-
 <?php
-if ($_SERVER["REQUEST_METHOD"] == "POST") {
-    // Retrieve form data
-    $email = $_POST['email'];
-    $household = (int) $_POST['household'];  // Cast to int for question 2
-    $meal_bag = $_POST['meal_bag'];
-    $name = $_POST['name'];
-    $address = $_POST['address'];
-    $phone = $_POST['phone'];
-    $photo_release = filter_var($_POST['photo_release'], FILTER_VALIDATE_BOOLEAN);  // Convert to boolean for question 7
-
-    // Validation for Question 2 (Household) and Question 7 (Photo Release)
-    $errors = [];
-
-    // Validate that household is a positive integer
-    if (!is_int($household) || $household <= 0) {
-        $errors[] = "Please enter a valid number for household members.";
-    }
+//start session and check login status
+session_cache_expire(30);
+session_start();
+
+$loggedIn = false;
+$accessLevel = 0;
+$userID = null;
+if (isset($_SESSION['_id'])) {
+    $loggedIn = true;
+    $accessLevel = $_SESSION['access_level'];
+    $userID = $_SESSION['_id'];
+}
 
-    // Validate that photo_release is a boolean (it should be true or false)
-    if ($photo_release === null) {
-        $errors[] = "Please select a valid option for photo release.";
-    }
+//allow only logged-in users to submit the form
+if (!$loggedIn || $accessLevel < 1) {
+    die("Access denied. Please log in to access the form.");
+}
 
-    // Check if all other required fields are filled
-    if (empty($email) || empty($meal_bag) || empty($name) || empty($address) || empty($phone)) {
-        $errors[] = "Please fill out all required fields.";
-    }
+//database connection
+$host = 'localhost';
+$dbname = 'odhsmd';
+$username = 'odhsmd';
+$password = 'odhsmd';
+
+try {
+    $conn = new PDO("mysql:host=$host;dbname=$dbname", $username, $password);
```

Comment:
We'll want to use the connect function from dbinfo.php instead of defining our database connection here. When we migrate to siteground our db credentials will change
```php
include_once('dbinfo.php');
$conn = connect();
```